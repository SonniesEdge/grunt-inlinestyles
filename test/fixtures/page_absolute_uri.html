<!DOCTYPE html>
<html>
  <head>
    <link rel="stylesheet" href="http://example.com/theme/css/main.css" data-inline="true">
  </head>
  <body>
    <h1>Test page</h1>
  </body>
</html>
