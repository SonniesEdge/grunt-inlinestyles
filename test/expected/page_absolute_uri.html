<!DOCTYPE html>
<html>
  <head>
    <style>
    h1 {
      font-style: italic;
    }
    </style>
  </head>
  <body>
    <h1>Test page</h1>
  </body>
</html>
